### DISCUSSION TITLE
SD Maid 2 startet die App Bereinigung immer wieder von vorne!

### DISCUSSION THREAD

**Discussion by Roger1465** [community/community#70628] [New to GitHub]
Sorry ich habe überhaupt keine Ahnung von dem hier und möchte lediglich ein Problem melden! Meine SD Maid 2, startet die App Bereinigung immer wieder von Neuem! Ich bin absolut Überfordert und weiß auch nach langem suchen nach dem Fehler nicht weiter.Schade ich ärgere mich,SD Maid 1nach Kauf, deinstalliert zu haben,denn mit Ihr hatte ich niemals ein Problem! Ich weiß zwar nicht was ich hier bei GitHub mache,hoffe aber man kann mir trotzdem helfen!? Gruß Roger✌🏻

**Comment by queenofcorgis** [comment]
Hi @Roger1465 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
